### ISSUE TITLE
Advertisements: ygosu.com [ Ad-Shield ]

### ISSUE THREAD

**Issue by piquark6046** [AdguardTeam/AdGuardExtra#283]
## Summary
On Mozilla Firefox with AdGuard Extra (Tampermonkey), advertisements on ygosu.com is not hidden.
If you cannot reproduce it, please scroll the page down.

## URLs
https://ygosu.com/
https://ygosu.com/community/all_article
https://ygosu.com/community/real_article/soccer/513517/?frombest=Y

## Screenshots
<details>
<summary>Screenshots</summary>

![Screenshot](https://user-images.githubusercontent.com/37463155/147848224-f23df8bf-13e6-416e-be7e-33fabfb5522a.png)
![Screenshot1](https://user-images.githubusercontent.com/37463155/147848228-5c785799-7618-465a-8f22-fa8e993ffdcf.png)

</details>

## Related Issues
https://github.com/AdguardTeam/AdGuardExtra/issues/258
https://github.com/AdguardTeam/AdguardFilters/issues/90634
https://github.com/AdguardTeam/AdguardFilters/issues/87725

**Comment by stanislav-atr** [comment]
@piquark6046, hi! These sliding out banners are only hidden on top of the screen by current fix.

**Comment by piquark6046** [comment]
@stanislav-atr 
Then, Cloud you please fix it for other AdGuard users?

**Comment by stanislav-atr** [comment]
@piquark6046, what do you mean by other AdGuard users? These might be removed in next fix iteration as i see it.

**Comment by piquark6046** [comment]
> what do you mean by other AdGuard users? 

I meant that What about update something as hiding the advertisements to make that a visitor cannot see the advertisements.

**Comment by piquark6046** [comment]
## maple.inven.co.kr
### URL
https://maple.inven.co.kr/

### Related Issue
https://github.com/AdguardTeam/AdguardFilters/issues/106383

**Comment by stanislav-atr** [comment]
@piquark6046, hi! I've updated ppss and ygosu, but was only able to reproduce ad-shield ads on mapple.inven once. Are these reproducible for you atm?

**Comment by piquark6046** [comment]
@stanislav-atr No. I cannot find ad-shield advertisements on maple.inven.co.kr today.
